lib/elixir/lib/module.ex

```diff
@@ -782,16 +803,17 @@ defmodule Module do
   ## Examples
 
       defmodule Example do
-        def version, do: 1
+        defmacro version, do: 1
         Module.definitions_in __MODULE__, :def  #=> [{:version, 0}]
-        Module.definitions_in __MODULE__, :defp #=> []
+        Module.definitions_in __MODULE__, :defmacro #=> []
```

Comment:
Why this change? If you change `version/0` to be a macro, then `Module.definitions_in(__MODULE__, :def)` will return `[]`, not `[{:version, 0}]`, and `:defmacro` will return `[{:version, 0}]` instead 😕